src/scenegraph/Thruster.cpp

```diff
@@ -100,21 +100,32 @@ namespace SceneGraph {
 			}
 		}
 		if (power < 0.001f) return;
+		
+		const float thrust_flicker = ((rand() % 10001) + 90000) / 100000.0f;
```

Comment:
I'd like a comment here to document the choice of these magic numbers. 
_same as comment below_